tests/testthat/test-ensembleMethods.R

```diff
@@ -176,22 +179,26 @@ context("Residual extraction")
 test_that("Residuals provided by residuals are proper for ensemble objects", {
   skip_on_cran()
   set.seed(2239)
-  ens.class <- caretEnsemble(models.class, trControl=trainControl(method="none"))
+  ens.class <- caretEnsemble(models.class, trControl = trainControl(method = "none"))
   models.subset <- models.reg[2:4]
   class(models.subset) <- "caretList"
-  ens.reg <- caretEnsemble(models.subset, trControl=trainControl(method="none"))
-  residTest <- residuals(ens.class)
-  residTest2 <- residuals(ens.reg)
+  ens.reg <- caretEnsemble(models.subset, trControl = trainControl(method = "none"))
+  suppressWarnings({
```

Comment:
What warnings does this give now?

Reply:
It isn't giving any warning. I suppose that I made a mistake.

Reply:
got it